modules/3_classification/src/classifier.cpp

```diff
@@ -60,4 +96,18 @@ void Classifier::classify(const cv::Mat& image, int k, std::vector<float>& proba
 
     // Copy output. "prob" is a name of output from .xml file
     float* output = req.GetBlob(outputName)->buffer();
+	
+	// Transfer output values to vector
+	std::vector<float> res;
+	for (int i = 0; i < req.GetBlob(outputName)->size(); i++)
+	{
+		res.push_back(output[i]);
+	}
+	topK(res, k, probabilities, indices);
+
+	// Normalize the output
+	for (int i = 0; i < k; i++)
+	{
+		probabilities[i] /= probabilities[0];
```

Comment:
I don't think that this is correct

Reply:
Unfortunately test is not reliable but this is output from https://travis-ci.com/github/dkurt/openvino_practice/builds/175027953:
```
[ RUN      ] classification.DenseNet
1.00000 streetcar, tram, tramcar, trolley, trolley car
14.91887 passenger car, coach, carriage
13.37559 electric locomotive
12.01191 trolleybus, trolley coach, trackless trolley
10.92209 school bus
```

Reply:
any suggestions?

Reply:
Try to apply SoftMax. At least we need to apply it somewhere :)

Reply:
ah, right, that is incorrect

Reply:
so, nothing changed as for me. Implemented `softmax` should be applied to outputs